### PR TITLE
update to the PR Owners responsibility

### PR DESCRIPTION
**Pull request by lalithvaka**
This PR will address issue: [2006](https://github.com/kubeflow/website/issues/2006)
Added "It is ultimately the PR creator's responsibility to see that the PR gets assigned to appropriate           reviewers and ping reviewers as needed" under "Quirks of the process / Reviewers, and approvers are unresponsive" section.

/assign @jlewi

### CODE REVIEW

File: content/en/docs/about/contributing.md [hunk 1]
```diff
@@ -258,6 +258,7 @@ is the state of today.
     their [PR dashboard](https://k8s-gubernator.appspot.com/pr)
   - An **author** can work around this by manually reading the relevant OWNERS files,
     `/unassign`'ing unresponsive individuals, and `/assign`'ing others
+  - It is ultimately the PR creator's responsibility to see that the PR gets assigned to appropriate           **reviewers** and ping **reviewers** as needed
```
Comment (jlewi) [inline content/en/docs/about/contributing.md:261]: This probably needs some additional context.

The PR creator should be responsible for escalating if need be and we should identify the appropriate escalation path

* e.g. ping the reviewers 
* If the reviewers don't respond look at the OWNERs file in root and ping approvers listed there
* If they don't respond email kubeflow-discuss

Comment (lalithvaka) [reply content/en/docs/about/contributing.md]: Thanks @jlewi , updated accordingly.

File: content/en/docs/about/contributing.md [hunk 2]
```diff
@@ -260,6 +260,9 @@ is the state of today.
     `/unassign`'ing unresponsive individuals, and `/assign`'ing others
   - This is a sign that our OWNERS files are stale; pruning the **reviewers** and **approvers** lists
     would help with this
+  - It is ultimately the PR **authors** responsibility to see that the PR gets assigned to appropriate    **reviewers** and follow an escalation path for review and approval if need be
```
[the remark this answers is not in the document]
Comment (lalithvaka) [reply content/en/docs/about/contributing.md]: Submitted the commit again to reflect the suggestion

**Pull request merged**